sephora.dme

```diff
@@ -1872,7 +1872,6 @@
 #include "code\modules\jobs\job_types\station_engineer.dm"
 #include "code\modules\jobs\job_types\virologist.dm"
 #include "code\modules\jobs\job_types\warden.dm"
-#include "code\modules\jobs\map_changes\map_changes.dm"
```

Comment:
Replace the pubby data in this fine with an include, Don't uninclude it.

Reply:
The file only exists for pubby, there's not much reason to keep it since it'd be a "blank" file that's entirely commented out. If that's fine with everyone then sure.

Reply:
Got added clarification from @francinum in discord. It was decided that he wanted the file even if it would be nothing but comments. Re-adding it per his request and with his comment suggestions.